test/extensions/tracers/zipkin/zipkin_tracer_impl_test.cc

```diff
@@ -466,6 +466,31 @@ TEST_F(ZipkinDriverTest, ZipkinSpanContextFromB3HeadersTest) {
   EXPECT_TRUE(zipkin_span->span().sampled());
 }
 
+TEST_F(ZipkinDriverTest, ZipkinSpanContextFromB3Headers128TraceIdTest) {
+  setupValidDriver();
+
+  const std::string trace_id =
```

Comment:
I'd actually test this.. below you aren't really testing this literally. For example, test that the high bits are in the right spot (so the input left hex characters are trace_id_high and right ones are trace_id)